ux-microservice/bin/www

```diff
@@ -80,11 +65,15 @@ function onError(error) {
 /**
  * Event listener for HTTP server "listening" event.
  */
-
 function onListening() {
-  var addr = server.address();
-  var bind = typeof addr === 'string'
-    ? 'pipe ' + addr
-    : 'port ' + addr.port;
-  debug('Listening on ' + bind);
+  const addr = server.address();
+  const bind = typeof addr === 'string' ? `pipe ${addr}` : `port ${addr.port}`;
+  debug(`Listening on ${bind}`);
```

Comment:
As this is the only (I think) use of `debug` as the logger, can we change this to `morgan` (or change `morgan` in the app to `debug`)?

Reply:
I want to install 'winston' as the logger, separate pr